projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,8 @@
+<cdk-virtual-scroll-viewport appendOnly itemSize="10" class="tabs-viewport">
+    <div class="virtual" *cdkVirtualFor="let c of allColors; let i = index" class="tabs-item">
+        <geoengine-color-attribute-input [ngModel]="c" (ngModelChange)="rebuildColorMap(i, $event)"></geoengine-color-attribute-input>
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/225277895-80f5c00a-ef9b-49b7-9a36-e6e0624152be.png)

Do you know what happens when I insert text?
At least I can click apply then… maybe we could prohibit this?